webhookClient/provide.go

```diff
@@ -0,0 +1,39 @@
+package webhookClient
+
+import (
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"go.uber.org/fx"
+)
+
+// PeriodicRegistererIn is an uber/fx parameter with the in information needed to create a new PeriodicRegisterer.
+type PeriodicRegistererIn struct {
+	fx.In
+	Registerer *BasicRegisterer
+	Interval   time.Duration `name:"periodic_registration_interval"`
+	Logger     log.Logger
+	Measures   *Measures
```

Comment:
Instead of `*Measures`, we can add `MeasuresIn` here (or potentially get rid of `MeasuresIn` if the name tagging is moved to `Measures` itself?) and then I don't think `NewProvideMeasures()` is needed below.